src/cwt.rs

```diff
@@ -0,0 +1,114 @@
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+use std::collections::BTreeMap;
+
+pub use crate::claim::{self, Claim, Label, NumericDate};
+
+/// CWT claims set.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq, Eq)]
+pub struct ClaimsSet(BTreeMap<Value, Value>);
+
+impl ClaimsSet {
+    pub fn insert_claim<T: Claim>(&mut self, claim: T) -> Option<Value> {
+        self.0.insert(T::label().into(), claim.into())
+    }
+
+    pub fn get_claim<T: Claim>(&self) -> Option<&Value> {
+        self.0.get(&T::label().into())
+    }
```

Comment:
I wonder whether these values should return `T` instead of `Value`?

For example, if you did something like:
```rust
let exp = claims_set.get_claim::<ExpirationTime>().unwrap();
```
`exp` would be a `Value`, so I need to process that and parse it into a `NumericDate` manually.

Reply:
I think for `get_claim` in particular yes, definitely see what you mean. Do you see a better way of going about than making `Claim`'s implement `TryFrom<Value> for Claim` as well, and then returning a `Result<Option<&T>, claim::Error>` from `get_claim`?